hawc_hal/healpix_handling/sparse_healpix.py

```diff
@@ -74,6 +73,38 @@ def __init__(self, partial_map, pixels_ids, nside, fill_value=UNSEEN):
 
         super(SparseHealpix, self).__init__(sparse=True, nside=nside)
 
+    def __add__(self, other_map):
+
+        # Make sure they have the same pixels
+        assert np.array_equal(self.pixels, other_map.pixels)
+
+        # inflate them
```

Comment:
There is no need to inflate them. You can just use `.as_partial` which returns a numpy array with the same shape as the selected pixels. So for example to do what you want to do you can do:

```python
# add them
partial_added = self.as_partial() + other_map.as_partial()
sparse_added = SparseHealpix(dense_added, self.pixels, self.nside)
```